rotkehlchen/accounting/cost_basis/base.py

```diff
@@ -364,12 +366,16 @@ def add_acquisition(self, acquisition: AssetAcquisitionEvent) -> None:
             self._acquisitions_heap,
             AssetAcquisitionHeapElement(self._count, acquisition),
         )
-        self.current_average_cost_basis += (acquisition.rate * acquisition.amount)
+        self.current_average_asset_value = (self.current_average_asset_value * self.remaining_amount + acquisition.amount * acquisition.rate) / (self.remaining_amount + acquisition.amount)  # noqa: E501
```

Comment:
Shouldn't the docstring be also adjusted. It seems to refer to this total ACB terminology which is super confusing.

Also if `current_average_asset_value` is `amount*rate` for all acquisitions then why multiply by remaining amount?

Reply:
`current_average_asset_value` is essentially the cost base, so it should not be multiplied by the amount

Reply:
so rename it to current_average_cost_basis or even `current_acb` so it's clear